### ISSUE TITLE
The training seems to stop？？？

### ISSUE THREAD

**Issue by limengfei3675** [signatrix/efficientdet#17]
Hello, thank you for your code. We ran the code and found that convergence was slow. What's more important is that the mAP on the training set will no longer increase to 0.34. This is the case for two consecutive days. The training seems to stop and it is useless to reduce the learning rate. What is going on in this situation? ? ? Stuck in a local optimum? ?

**Comment by uvipen** [comment]
Hi,
Which gpus you are using? We trained for 9 days on 3 GTX 1080 Ti

**Comment by limengfei3675** [comment]
Thanks, and can you tell me final epoch loss???

**Comment by uvipen** [comment]
Unfortunately, we dont keep the tensorboard for it, but if we remember correctly it is about 0.45 for test set
